simpledotorg/simple-server
# Replace "Protocols" copy with "Medications"

**Story card:** [ch3456](https://app.clubhouse.io/simpledotorg/story/3456/replace-protocols-copy-with-medications)

## Because

Using "Protocol" is misleading. "Protocols" are customizable lists of medications that appear in the Simple App. Using "Medications" is much clearer.

## This addresses

Updates all "protocols" copy in the "Protocol medications" admin pages

## Test instructions

Check out [this Figma file](https://www.figma.com/file/9inSvRD4dXcMfXchlyDZVV/Dashboard-Production?node-id=5989%3A2973) for a full list of copy changes
